web-server/routes/calendar.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+const loadDigest = require('../../digest')
+
+describe('GET /calendar route', function() {
+  it('Renders a page with iframe containing calendar', function(done){
+    webServer()
```

Comment:
please change from using the `done callback` async mocha test style, to using the `return a promise` style. Also see the new tools. you can now do `return this.get('/calendar/')` :)

Reply:
Can you point me at a good example? I tried to replicate the style of goals.test.js